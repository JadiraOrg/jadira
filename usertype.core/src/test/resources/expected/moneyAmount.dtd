<!ELEMENT dataset (
    MONEYAMOUNT*)>

<!ELEMENT MONEYAMOUNT EMPTY>
<!ATTLIST MONEYAMOUNT
    ID CDATA #REQUIRED
    MONEY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

