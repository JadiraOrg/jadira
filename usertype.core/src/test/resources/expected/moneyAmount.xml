<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <moneyAmount ID="0" MONEY="100.00" NAME="test_0"/>
  <moneyAmount ID="1" MONEY="100.10" NAME="test_1"/>
  <moneyAmount ID="2" MONEY="0.99" NAME="test_2"/>
  <moneyAmount ID="3" NAME="test_3"/>
</dataset>
